calico_containers/calicoctl.py

```diff
@@ -437,6 +461,64 @@ def normalize_version(version):
     """
     return [int(x) for x in re.sub(r'(\.0+)*$','', version).split(".")]
 
+
+def get_host_ips():
```

Comment:
ip route is structured text; you shouldn't need a regex.  Just split the output and use `netaddr` to parse addresses.